test/nimble_template/addons/credo_test.exs

```diff
@@ -28,6 +28,7 @@ defmodule NimbleTemplate.Addons.CredoTest do
                    defp deps do
                      [
                        {:credo, "~> 1.4", [only: [:dev, :test], runtime: false]},
+                       {:compass_credo_plugin, "~> 1.0.0", [only: [:dev, :test], runtime: false]},
```

Comment:
Should we avoid the call to Hex.pm to get the `compass_credo_plugin` version by using the `@describetag mock_latest_package_versions: [{:compass_credo_plugin, "1.0.0"}]` @hanam1ni

Reply:
I'm opening a PR for this issue as well FYI @hanam1ni https://github.com/nimblehq/elixir-templates/pull/284

Reply:
@andyduong1920 Good catch 👍 I updated in [a7416d4](https://github.com/nimblehq/elixir-templates/pull/276/commits/a7416d412209f50c9167b42fb14970a0ec7a4847)